src/test/java/io/debezium/connector/spanner/BasicSanityCheckIT.java

```diff
@@ -0,0 +1,106 @@
+        Configuration config = Configuration.create()
+                .with("gcp.spanner.instance.id", database.getInstanceId())
+                .build();
+        start(SpannerConnector.class, config, (success, msg, error) -> {
+            assertThat(success).isFalse();
+            assertThat(msg.contains("Connector configuration is not valid"));
+        });
+        assertConnectorNotRunning();
+    }
+
+    @Test
+    public void shouldNotStartConnectorWithoutNonExistentChangeStreams() throws InterruptedException {
+        final Configuration config = Configuration.copy(baseConfig)
+                .with("gcp.spanner.change.stream", "fooBar")
+                .with("name", tableName + "_test")
+                .with("gcp.spanner.start.time",
+                        DateTimeFormatter.ISO_INSTANT.format(Instant.now()))
+                .build();
+        start(SpannerConnector.class, config, (success, msg, error) -> {
+            assertThat(success).isFalse();
+        });
+        assertConnectorNotRunning();
+    }
+
+    @Test
+    public void shouldNotStartConnectorWithOutOfRangeHeartbeatMillis() throws InterruptedException {
+        final Configuration config = Configuration.copy(baseConfig)
+                .with("gcp.spanner.change.stream", changeStreamName)
+                .with("heartbeat.interval.ms", "1")
+                .with("gcp.spanner.start.time",
+                        DateTimeFormatter.ISO_INSTANT.format(Instant.now().plus(2, ChronoUnit.DAYS)))
+                .build();
+        start(SpannerConnector.class, config, (success, msg, error) -> {
+            assertThat(success).isFalse();
+            assertThat(msg.contains("Heartbeat interval must be between 100 and 300000"));
+        });
+        assertConnectorNotRunning();
+    }
+
+    @Test
+    public void shouldStreamUpdatesToKafka() throws InterruptedException {
+        System.out.println("test hahahaha");
+        final Configuration config = Configuration.copy(baseConfig)
+                .with("gcp.spanner.change.stream", changeStreamName)
+                .with("name", tableName + "_test")
+                .with("gcp.spanner.start.time",
+                        DateTimeFormatter.ISO_INSTANT.format(Instant.now()))
+                .build();
+        initializeConnectorTestFramework();
+        start(SpannerConnector.class, config);
+        assertConnectorIsRunning();
+        databaseConnection.executeUpdate("insert into " + tableName + "(id, name) values (1, 'some name')");
+        databaseConnection.executeUpdate("update " + tableName + " set name = 'test' where id = 1");
+        databaseConnection.executeUpdate("delete from " + tableName + " where id = 1");
+        waitForCDC();
+        SourceRecords sourceRecords = consumeRecordsByTopic(10, false);
+        List<SourceRecord> records = sourceRecords.recordsForTopic(getTopicName(config, tableName));
+        assertThat(records).hasSize(4); // insert + update + delete + TOMBSTONE
```

Comment:
Is there a way to do a quick verification of the record contents? I.e. just verify the record mod types

Reply:
Done. Added verification to check mod types are in the order of create -> update -> delete -> tombstone.